deployments/docker/README.md

```diff
@@ -57,7 +57,7 @@ set the following environment variables on the agent container:
 | Environment Variable  | Required | Description |
 | --------- | -------- | ----------- |
 | `SFX_ACCESS_TOKEN` | **yes** | The SignalFx API access token. |
-| `SFX_INGEST_URL` | no | Often used in conjunction with the [SignalFx Metric Proxy](https://github.com/signalfx/metricproxy) to specify a different target URL for datapoints and events. If not specified, this defaults to the global SignalFx ingest server. |
+| `SFX_INGEST_URL` | no | Often used in conjunction with the [SignalFx Gateway](https://github.com/signalfx/metricproxy) to specify a different target URL for datapoints and events. If not specified, this defaults to the global SignalFx ingest server. |
```

Comment:
@barbara-sfx do we know the new github repo path yet?  That should probably be changed too.